[2024-01-28T15:36:17.121+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-02-14T00:00:00+00:00 [queued]>
[2024-01-28T15:36:17.144+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-02-14T00:00:00+00:00 [queued]>
[2024-01-28T15:36:17.146+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-28T15:36:17.204+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2023-02-14 00:00:00+00:00
[2024-01-28T15:36:17.222+0000] {standard_task_runner.py:60} INFO - Started process 2437 to run task
[2024-01-28T15:36:17.231+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'scheduled__2023-02-14T00:00:00+00:00', '--job-id', '1234', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmpe_zdt73z']
[2024-01-28T15:36:17.238+0000] {standard_task_runner.py:88} INFO - Job 1234: Subtask get_stackoverflow_data
[2024-01-28T15:36:17.349+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data scheduled__2023-02-14T00:00:00+00:00 [running]> on host 9924936b75b5
[2024-01-28T15:36:17.563+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2023-02-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-02-14T00:00:00+00:00'
[2024-01-28T15:36:18.285+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-28T15:36:18.316+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20230214T000000, start_date=20240128T153617, end_date=20240128T153618
[2024-01-28T15:36:18.488+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-28T15:36:19.525+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
